o2.sh

```diff
@@ -25,6 +25,7 @@ requires:
   - cgal
   - KFParticle
   - VecGeom
+  - FLUKA_VMC
```

Comment:
I think the dependency is correct but why is it added here instead of simulation.sh?
This should be the correct place.

Reply:
Ok, I will have a look and make a new PR.

Reply:
You are right, my bad fixed in #3005.